docs/src/index.md

```diff
@@ -184,5 +184,3 @@ Finally, all `Content` subclasses can be converted with the following:
 * `AwkwardArray.to_vector`: converts an Awkward Array into Julia data
 * `AwkwardArray.from_buffers`: constructs an Awkward Array from a Form (JSON), length, and buffers for zero-copy passing from Python
 * `AwkwardArray.to_buffers`: deconstructs an Awkward Array into a Form (JSON), length, and buffers for zero-copy passing to Python
-
-_(This will turn into proper documentation, eventually.)_
```

Comment:
@jpivarski - the idea is to trim the top README.md file and have full documentation here that is generated as a documentation by Documeter.jl. The full infrastructure is already in place. The dev version of documentation is at https://juliahep.github.io/AwkwardArray.jl/dev/